### ISSUE TITLE
No login prompt

### ISSUE THREAD

**Issue by DevSlashNull** [keybase/keybase-issues#2558]
The Keybase app has stopped prompting me for my login passphrase. When my machine starts up, the app is already open and waiting on my desktop.

Running Lubuntu 16.04.

**Comment by 0x4445565A** [comment]
Are you able to still use your keys?  Is everything unlocked?  I know there is a push to make it so you aren't required passwords every login, maybe it is already in production?

**Comment by DevSlashNull** [comment]
Yes, I'm able to use my keys, and yes - everything is unlocked. No password requirement is a terrible idea - so anyone who gets their hands on my laptop has immediate access to my keys and folders?

**Comment by malgorithms** [comment]
@DevSlashNull  - can you confirm whether you do/don't have a PGP private key imported in keybase's keychain? You can check with `keybase pgp purge` (which doesn't actually purge unless you pass `-p`)

If you don't, it'll print something like this:
```
> keybase pgp purge
No PGP keys found in local keyring
```

If you do have one, it will export your PGP key, encrypted with `keybase encrypt`.

I ask because as @0x4445565A mentioned, we've gotten a lot of feedback that by switching to per-device keys, we should take the stance that a device key should rely on a device's storage, and it's more annoying to have to enter passwords just to unlock a key unique to that device. (Distinct, say, from a key you use on multiple machines.)  On the other hand, if you have a PGP key that can be accessed by your keybase key, you're supposed to get a dialog explaining this new key model, and explaining that if you don't like it, you can pull your PGP key from keybase's key ring. If you do have a PGP key AND didn't see this, then we have a bug.

**Comment by DevSlashNull** [comment]
"No PGP keys found in local keyring"

This is a fresh install of Keybase on my secondary machine.

**Comment by malgorithms** [comment]
ok, thanks.  This matches our newer model mentioned above. The idea is that whoever can access your files on your  laptop should have access to your laptop's device key; no cross-device keys should be exposed. Basically that a device key is as strong as that device's storage.

**Comment by DevSlashNull** [comment]
Will this be an optional feature? Being slightly more paranoid than the average person, I'd prefer the inconvenience/added security of being prompted for my passphrase every time.

**Comment by malgorithms** [comment]
we're talking about it. in fact, we were just saying how we took your comment as a vote for it as a feature.

**Comment by DevSlashNull** [comment]
Excellent - I was a little alarmed when I turned on my machine  and the app was waiting on my desktop with no authentication prompt. Other than that minor concern, you guys are doing great!
